src/maintainer/knowledge_base.rst

```diff
@@ -554,45 +554,96 @@ Special packages
 ================
 
 BLAS
-----
+....
 
-.. todo::
+If a package needs one of BLAS, CBLAS, LAPACK, LAPACKE, use the following in the
+host of the recipe,
+
+.. code-block:: yaml
+    requirements:
+      host:
+        - libblas
+        - libcblas
+        - liblapack
+        - liblapacke
+
+.. note::
+You should specify only the libraries the package needs. (i.e. if the package
+doesn't need LAPACK, remove liblapack and liblapacke)
+
+At recipe build time, above requirements would download the NETLIB's reference
+implementations and build your recipe against those.
+At runtime by default the following packages will be used,
+
+.. code-block:: yaml
+    - openblas   # [not win]
+    - mkl        # [win]
+
+If a package needs a specific implementation's internal API for more control you can have,
+
+.. code-block:: yaml
+    requirements:
+      host:
+        - {{ blas_impl }}
+      run:
+        - libblas * *{{ blas_impl }}
+        - {{ blas_impl }}
+
+This would give you a matrix builds for different blas implementations. If you only want to support
+a specific blas implementation,
+
+.. code-block:: yaml
+    requirements:
+      host:
+        - openblas
+      run:
+        - libblas * *openblas
+        - openblas
+
+.. note::
+`blas_*` features should not be used anymore.
+
+Switching BLAS implementation
+"""""""""""""""""""""""""""""
+
+You can switch your BLAS implementation by doing,
 
-  The information regarding the blas package may now be outdated and needs checking.
+.. code-block:: bash
+    conda install "libblas=*=*mkl"
+    conda install "libblas=*=*openblas"
+    conda install "libblas=*=*blis"
+    conda install "libblas=*=*netlib"
+
+This would change the BLAS implementation without changing the conda packages depending
+on BLAS.
 
-BLAS metapackage
-................
+Following legacy commands are also supported as well.
 
-   -  Version will have two values X.Y
+.. code-block:: bash
+    conda install "blas=*=mkl"
+    conda install "blas=*=openblas"
+    conda install "blas=*=blis"
+    conda install "blas=*=netlib"
 
-      -  X represents changes to the metapackage.
-      -  Y represents priority of BLAS (if we change priorities BLASes X
-         must be bumped, if we want to prioritize something new over
-         OpenBLAS we do not need to change X)
+How it works
+""""""""""""
 
-         -  1 is OpenBLAS
-         -  0 is None (no BLAS whatsoever)
+At recipe build time, the netlib packages are used. This means that the downstream package will
+link to ``libblas.so.3`` in the ``libblis=*=*netlib` and will use only the reference
```

Comment:
"blis" is on the lhs of the `{package}={version}={buildstr}` here and in the paragraph around line 640; whereas, I'm accustomed to seeing it in the buildstr slot. Typo or something I should learn about?

Reply:
Should have been `libblas` instead of `libblis`. Thanks